src/v2/Apps/Artwork/Components/TrustSignals/TrustSignal.tsx

```diff
@@ -19,14 +19,16 @@ export const TrustSignal: FC<TrustSignalProps> = ({
     <Flex {...other}>
       {Icon}
       <Flex flexDirection="column" ml={1}>
+        {/* FIXME: Remove this lint ignore */}
+        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
         <Link onClick={onClick}>
```

Comment:
Hm, I wonder if we could just `Link as="button"` with these actually; since they don't actually have hrefs.

Reply:
I can try!